doc/examples/brain_extraction_dwi.py

```diff
@@ -22,6 +23,12 @@
 models. For this example, the data comes from a 1.5 Tesla Siemens MRI.
 """
 
+
+from dipy.data import get_fnames
+from dipy.io.image import load_nifti, save_nifti
+from dipy.segment.mask import median_otsu
```

Comment:
the block above appears twice and is duplicated with the import on the top

Reply:
okay got it.